coderd/activitybump_internal_test.go

```diff
@@ -92,117 +101,120 @@ func Test_ActivityBumpWorkspace(t *testing.T) {
 		},
 	} {
 		tt := tt
-		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
-
-			var (
-				now   = dbtime.Now()
-				ctx   = testutil.Context(t, testutil.WaitShort)
-				log   = slogtest.Make(t, nil)
-				db, _ = dbtestutil.NewDB(t)
-				org   = dbgen.Organization(t, db, database.Organization{})
-				user  = dbgen.User(t, db, database.User{
-					Status: database.UserStatusActive,
-				})
-				_ = dbgen.OrganizationMember(t, db, database.OrganizationMember{
-					UserID:         user.ID,
-					OrganizationID: org.ID,
-				})
-				templateVersion = dbgen.TemplateVersion(t, db, database.TemplateVersion{
-					OrganizationID: org.ID,
-					CreatedBy:      user.ID,
-				})
-				template = dbgen.Template(t, db, database.Template{
-					OrganizationID:  org.ID,
-					ActiveVersionID: templateVersion.ID,
-					CreatedBy:       user.ID,
-				})
-				ws = dbgen.Workspace(t, db, database.Workspace{
-					OwnerID:        user.ID,
-					OrganizationID: org.ID,
-					TemplateID:     template.ID,
-					Ttl:            sql.NullInt64{Valid: true, Int64: int64(tt.workspaceTTL)},
-				})
-				job = dbgen.ProvisionerJob(t, db, database.ProvisionerJob{
-					OrganizationID: org.ID,
-					CompletedAt:    tt.jobCompletedAt,
-				})
-				_ = dbgen.WorkspaceResource(t, db, database.WorkspaceResource{
-					JobID: job.ID,
+		for _, tz := range timezones {
```

Comment:
QQ: Most likely I don't see the issue, but is it mandatory to run all tests against all timezones? I'm concerned about increasing the total number of tests which might be just mirrors of each other (no value added).

Reply:
No, it doesn't make sense for a unit test that does not depend on timezones to run against multiple timezones.

Reply:
Edit: now that I read a bit more, I'm trying to cover the following cases:
- DST versus non-DST
- Positive versus negative offsets
- Fractional vs whole-hour offsets
- and UTC of course as a baseline

These tests run very quickly as we don't spin up an entire coderd, and given the bug that existed I'd rather have them than not.

Reply:
That's a good justification, maybe leave a comment somewhere around the test, and describe cases?

Reply:
Done!